### ISSUE TITLE
Serde support

### ISSUE THREAD

**Issue by polachok** [capnproto/capnpc-rust#47]
C++ capnproto implementation supports json serialization. Would you be interested in adding support for serde? I think a [Serialize](https://serde.rs/impl-serialize.html) impl could be generated by capnpc.

**Comment by dwrensha** [comment]
Yeah, JSON conversion would be useful! The C++ implementation makes use of the [dynamic reflection](https://capnproto.org/cxx.html#dynamic-reflection) API. I think something similar would be a good way forward in capnproto-rust.
